knees4bees/rancid-tomatillos
# Feature/update error handling

## Is this a fix, feature, or refactor?
Fix and refactor

## What is the change?
* Refactors error handling to work with Router
* Adds testing for failed and not-ok network requests
* Minor cleanup

## What does it fix?
* Fixes problem with error handling where error message was not displaying for failed network request on movie detail page.

## Where should the reviewer start?
Most of the interesting changes are in `App.js` and `Error.js`.

## How should this be tested?
Fetch this branch and then 
```
npm i -D cypress
npm run test
```
Confirm that all tests are passing.


## Relevant ScreenShots
